src/app/views/templates/sms-verification-warning.view.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+  <head> </head>
+  <body>
+    <p>Dear form admin(s),</p>
+    <p>
+      You are receiving this as you have enabled SMS OTP verification on a
+      Mobile Number field of your form:
+      <a href="<%= formLink %>"> '<%= formTitle %>' </a>.
+    </p>
+    <p>
+      You can receive <b><%= numAvailable %></b> more responses until free SMS
+      OTP verification is automatically disabled.
+    </p>
+    <p>
+      If you require SMS OTP verification for more than 10,000 responses, please
```

Comment:
Should this be `smsVerificationLimit` instead of 10,000?

Reply:
thx, good catch!